packages/connector-jsdom/src/connector.ts

```diff
@@ -288,9 +288,6 @@ export default class JSDOMConnector implements IConnector {
     public close() {
         try {
             this._window.close();
```

Comment:
Did you figured out why this coud be `undefined`?

Reply:
Yes, if the resource is not an HTML or there is an error download the website, the `this._window` is not initialized, but we are calling to close anyways.